.github/workflows/run_test_from_pr.yml

```diff
@@ -0,0 +1,55 @@
+name: Run (SLOW) desired tests on our runner from a PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number: 
+        description: 'PR number'
+        required: true
+      docker_image:
+        default: 'diffusers/diffusers-pytorch-cuda'
+        description: 'Name of the Docker image'
+        required: true
+      test_command:
+        description: 'Test command to run (e.g.: `pytest tests/pipelines/dit/`). Any valid pytest command can be provided.'
+        required: true
+
+env:
+  IS_GITHUB_CI: "1"
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  HF_HOME: /mnt/cache
+  DIFFUSERS_IS_CI: yes
+  OMP_NUM_THREADS: 8
+  MKL_NUM_THREADS: 8
+  RUN_SLOW: yes
+
+jobs:
+  run_tests:
+    name: "Run a test on our runner from a PR"
+    runs-on: [single-gpu, nvidia-gpu, "t4", ci]
+    container:
+      image: ${{ github.event.inputs.docker_image }}
+      options: --gpus all --privileged --ipc host -v /mnt/cache/.cache/huggingface:/mnt/cache/
+
+    steps:
+      - name: NVIDIA-SMI
+        run: |
+          nvidia-smi
+
+      - name: Install `gh`
+        run: | 
+            (type -p wget >/dev/null || (apt update && apt-get install wget -y)) \
```

Comment:
I guess this is based on: https://github.com/cli/cli/blob/trunk/docs/install_linux.md#official-sources, maybe could be added as a comment.